includes/server-timing/hooks.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Hook callbacks used for Server Timing.
+ *
+ * @package performance-lab
+ *
+ * @since n.e.x.t
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+
+/**
+ * Adds server timing to REST API response.
+ *
+ * @since n.e.x.t
+ *
+ * @param WP_REST_Response|WP_Error $response Result to send to the client. Usually a `WP_REST_Response`.
+ * @return WP_REST_Response|WP_Error Filtered response.
+ */
+function rest_post_dispatch_add_server_timing( $response ) {
+	if ( ! defined( 'REST_REQUEST' ) || ! REST_REQUEST || ! function_exists( 'perflab_server_timing' ) || ! $response instanceof WP_REST_Response ) {
```

Comment:
This line is fairly long, consider breaking into multiple lines. I'm also not certain that the `function_exists()` check is needed, since this function is defined in the same plugin.

```suggestion
if ( 
    ! defined( 'REST_REQUEST' ) || ! REST_REQUEST
    || ! $response instanceof WP_REST_Response
) {
```